packages/wasm-terminal/package.json

```diff
@@ -38,15 +35,15 @@
     "dom",
     "browser"
   ],
-  "author": "Aaron Turner",
+  "author": "Wasmer Engineering Team <engineering@wasmer.io>",
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/wasmerio/wasmer-js/issues"
   },
   "homepage": "https://github.com/wasmerio/wasmer-js/tree/master/packages/wasm-terminal",
   "dependencies": {
-    "@wasmer/wasi": "^0.3.2",
-    "@wasmer/wasmfs": "^0.3.2",
+    "@wasmer/wasi": "file:../wasi",
```

Comment:
How come you changed this? The reason this is the way it is, is so that lerna can manage the version and how they are installed for local dependent packages 😯

If you are having trouble with the package versions not resolving correctly, run `npm install` at the root, or even better, run `npm run lerna:link` 😄